com.unity.shadergraph/Editor/Importers/ShaderSubGraphImporter.cs

```diff
@@ -292,7 +292,10 @@ static void ProcessSubGraph(SubGraphAsset asset, GraphData graph, ShaderGraphImp
                 foreach (var child in category.Children)
                 {
                     var prop = propertiesList.Find(p => p.guid == child.guid);
-                    orderedProperties.Add(prop);
+                    // Not all properties in the category are actually on the graph.
+                    // In particular, it seems as if keywords are not properties on sub-graphs.
+                    if(prop != null)
```

Comment:
I noticed this while testing, but a null exception would happen accessing `orderedProperties` below because this `Find` can return null. I believe that sub-graphs don't put Keywords in `propertyList`.

Reply:
I think this "if" is your formatting error.